code/edgeidentity/src/test/java/com/adobe/marketing/mobile/edge/identity/IdentityExtensionTests.java

```diff
@@ -629,39 +629,41 @@ public void test_processCachedEvents_processesWhenBooted() {
 	// ========================================================================================
 	// handleRequestContent
 	// ========================================================================================
-	// use callback to see if sharedstate is set properly
-	// similar to other ones where update identities
-	// if calling updateAdId directly, already passing in a callback
+
 	@Test
 	public void test_handleRequestContent_processesWhenBooted() {
```

Comment:
Is this change intended or is it a result of the rebase? It seems off when comparing with the feature branch - https://github.com/adobe/aepsdk-edgeidentity-android/blob/feature/adIdConsent/code/edgeidentity/src/test/java/com/adobe/marketing/mobile/edge/identity/IdentityExtensionTests.java#L634

Reply:
Should be fixed with latest adIdConsent merge! Thanks for pointing this out